xml/sle_update_online.xml

```diff
@@ -17,16 +17,15 @@
   </dm:docmanager>
   <abstract>
    <para os="sles;sled">
-    &suse; offers an intuitive graphical and a simple command line tool to
-    upgrade a running system to a new service pack. They provide support for
-    <quote>rollback</quote> of service packs and more. This chapter explains
-    how to do a service pack upgrade step by step with these tools.
+    &suse; offers an intuitive graphical and a simple command-line tool to
```

Comment:
It's `command line`, not `command-line`. Please consult [our Style Guide](https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-terminology) if you are unsure.
```suggestion
    &suse; offers an intuitive graphical and a simple command line tool to
```

Reply:
The style guide only defines the _noun_ "command line". It says nothing about the use of "command-line" as an attribute in front of another noun (i.e. "tool" in this case). Afaik, "command-line tool" is more grammatically and thus should be preferable to "command line tool", even if the Style Guide does not go into that kind of detail.

Reply:
In this case, `command line` is not a noun but a compound adjective. Hence command-line tool.

Reply:
> an attribute in front of another noun

We call them _adjectives_. :smile: Sorry, I couldn't help it!

Reply:
Ok then, marking as resolved.